js/object.js

```diff
@@ -0,0 +1,30 @@
+const todos = {
+  contents: ['机を片付ける', '牛乳を買う', '散歩する'],
+  genre: ['掃除', '買い物', '運動'],
+};
+todoList(todos);
+addTodo();
+todoList(todos);
+
+function addTodo() {
+  const contents = prompt('タスクを入力してください。');
+  if (contents) {
+    const genre = prompt('タスクを追加しました。続いてジャンルを入力してください。');
+    if (genre) {
+      todos.contents.push(contents);
+      todos.genre.push(genre);
+      alert('ジャンルを追加しました。');
+    } else {
+      alert('ジャンルを追加できませんでした。やり直してください。');
+    }
+  } else {
+    alert('タスクを追加できませんでした。やり直してください。');
+  }
+}
+
+function todoList(todos) {
+  console.log('========================\n 現在持っているタスク一覧\n ========================')

```

Comment:
Space が入ってしまっていて、表示時にちょっとずれてます。
```suggestion
  console.log('========================\n 現在持っているタスク一覧\n========================')

```